src/mscorlib/src/System/IO/StreamReader.cs

```diff
@@ -529,6 +532,22 @@ private bool IsPreamble()
             return _checkPreamble;
         }
 
+        private StringBuilder AcquireStringBuilder(int capacity)
```

Comment:
It's likely worth naming this something like GetSharedStringBuilder or AcquireSharedStringBuilder to make it more clear in the few methods that call it that the StringBuilder object is shared.